src/mutator.cc

```diff
@@ -470,9 +470,18 @@ void Mutator::RegisterPostProcessor(const protobuf::Descriptor* desc,
 void Mutator::ApplyPostProcessing(Message* message) {
   const Descriptor* descriptor = message->GetDescriptor();
 
-  auto it = post_processors_.find(descriptor);
-  if (it != post_processors_.end()) {
-    it->second(message, random_());
+  size_t num_callbacks = post_processors_.count(descriptor);
+  if (num_callbacks > 0) {
+    size_t chosen_callback_index = random_() % num_callbacks;
+    auto it = post_processors_.equal_range(descriptor);
+    size_t index = 0;
+    for (auto callback = it.first; callback != it.second; ++callback) {
+      if (index == chosen_callback_index) {
+        callback->second(message, random_());
+        break;
```

Comment:
Ensures that we don't call more than one postprocessor per descriptor type in case there are multiple postprocessors registered for the said descriptor type.

Reply:
I think we should call all callbacks.
One of use-cases for them is to enforce particular constraints on some fields. Different callback could be responsible for different parts of the message, but we still need to call all of them to make it work,
If different random behavior is needed then user can randomly branch it in the callback itself.

Actually I think that multiple callbacks per type is also unnecessary, as user can branch inside of callback as well,
However it's trivial to support them and this behavior is probably less confusing than invoking only the last one.

Reply:
Sounds reasonable to me. Is there a suggested way to use `_seed` provided by postprocessor? I would like to use the `_seed` to seed a new RNG that can be repeated queried to generate new yet deterministic random numbers inside callbacks.

Reply:
Correct, seed is intended to be used for any RNG related needs, e.g. init std::rand